src/plugin.ts

```diff
@@ -79,7 +89,10 @@ const getCSS = (config: RealtimeColorOptions) => {
   const modifiers = availableModifiers[config.shadeAlgorithm];
   const variables: Record<string, string> = {};
   const altVariables: Record<string, string> = {};
-  for (const [colorName, color] of Object.entries(colors)) {
+
+  for (const [colorName, color] of Object.entries(colors).filter(
+    ([_, color]) => color === "dynamic()",
```

Comment:
Shouldn't it be `color !== "dynamic()"`?
Otherwise it would generate the colors for all **dynamic** colors?

Reply:
lol, good catch, maybe I removed the `!` before pushing